contracts/neutron_interchain_queries/src/testing/tests.rs

```diff
@@ -59,6 +61,33 @@ fn build_registered_query_response(
     )
 }
 
+fn build_registered_tx_query_response(
```

Comment:
This function is very similar with `build_registered_kv_query_response`. Can't we use one method for both query responses (kv and tx)?